example/spec/helpers/configUtils.js

```diff
@@ -1,32 +1,7 @@
 'use strict';
 
-const fs = require('fs-extra');
```

Comment:
Now that we are using "wait for s3 object" instead of "wait for cloud formation update", and we are also no longer trying to do a "delete workflow during execution" test, there was a lot of unused code that could be pruned out.

Reply:
👍